src/test/java/uk/gov/ons/census/action/model/repository/CaseRepositoryIT.java

```diff
@@ -133,7 +162,7 @@ private void setupTestCases(int caseCount, boolean receipted) {
   }
 
   private Specification<Case> createSpecificationForUnreceiptedCases() {
```

Comment:
We should probably rename this to `createSpecificationForActionableCases` like we did in ActionRuleProcessor.java